python/ray/serve/global_state.py

```diff
@@ -88,83 +128,24 @@ class GlobalState:
         2. A actor supervisor service
     """
 
-    def __init__(self, master_actor_handle=None):
-        # Get actor nursery handle
-        if master_actor_handle is None:
-            master_actor_handle = ray.util.get_actor(SERVE_MASTER_NAME)
-        self.master_actor_handle = master_actor_handle
+    def __init__(self, master_actor=None):
+        # Get actor nursery handle.
+        if master_actor is None:
+            master_actor = ray.util.get_actor(SERVE_MASTER_NAME)
+        self.master_actor = master_actor
 
-        # Connect to all the table
-        bootstrap_config = ray.get(
-            self.master_actor_handle.get_bootstrap_state_dict.remote())
-        kv_store_connector = bootstrap_config[BOOTSTRAP_KV_STORE_CONN_KEY]
+        # Connect to all the tables.
+        kv_store_connector = ray.get(
+            self.master_actor.get_kv_store_connector.remote())
         self.route_table = RoutingTable(kv_store_connector)
```

Comment:
do we still need the tables in GlobalState?

Reply:
They'll be removed in follow-up